### PR TITLE
robin created pit scout, not completed

### PR DESCRIPTION
**Pull request by rkang1357**

### CODE REVIEW

File: integrated-scouting-apps/src/public/stylesheets/style.css [hunk 1]
```diff
@@ -28,6 +28,7 @@ body {
     padding-right: 5%;
     color: white;
 }
+
```
Comment (nk6432) [inline integrated-scouting-apps/src/public/stylesheets/style.css:31]: remove extra line

File: integrated-scouting-apps/src/public/stylesheets/style.css [hunk 2]
```diff
@@ -161,6 +162,60 @@ input[type="checkbox"], .ant-checkbox-inner {
     color: white !important;
     text-align: center !important;
 }
+.pitInput {
```
Comment (nk6432) [inline integrated-scouting-apps/src/public/stylesheets/style.css:165]: lowercase

File: integrated-scouting-apps/src/public/stylesheets/style.css [hunk 3]
```diff
@@ -161,6 +162,60 @@ input[type="checkbox"], .ant-checkbox-inner {
...
+    width: 20%;
+    margin-bottom: 6%;
+    min-width: 35%;
+    height: 25% !important;
+    min-height: 25% !important;
+    border: 4px solid white !important;
+    border-radius: 25px;
+    background-color: #32a7dc;
```
Comment (nk6432) [inline integrated-scouting-apps/src/public/stylesheets/style.css:173]: change to transparent for all background color

File: integrated-scouting-apps/src/routes/pit.tsx [hunk 4]
```diff
@@ -1,282 +1,128 @@
 import '../public/stylesheets/style.css';
 import logo from '../public/images/logo.png';
+import back from '../public/images/back.png';
 import { Checkbox, Flex, Form, Input, InputNumber, Select, Tabs, TabsProps, Upload, message } from 'antd';
 import { useEffect, useState } from 'react';
 import TextArea from 'antd/es/input/TextArea';
 import { RcFile, UploadChangeParam, UploadFile, UploadProps } from 'antd/es/upload/interface';
 import { LoadingOutlined, PlusOutlined } from '@ant-design/icons';
+import '../public/stylesheets/style.css';
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/pit.tsx:9]: remove

File: integrated-scouting-apps/src/routes/pit.tsx [hunk 5]
```diff
@@ -1,282 +1,128 @@
...
+import no_image from '../public/images/no_image.png';
+import field_blue from '../public/images/field_blue.png';
+import field_red from '../public/images/field_red.png';
+import { useRef} from 'react';
+import {Button } from 'antd';
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/pit.tsx:14]: remove curly braces

File: integrated-scouting-apps/src/routes/pit.tsx [hunk 6]
```diff
@@ -1,282 +1,128 @@
...
+import { ReactSketchCanvas, ReactSketchCanvasRef } from 'react-sketch-canvas';
+import { type } from '@testing-library/user-event/dist/type';
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/pit.tsx:16]: whars this?

File: integrated-scouting-apps/src/routes/pit.tsx [hunk 7]
```diff
@@ -1,282 +1,128 @@
...
 
 function PitScout(props: any) {
-  const [form] = Form.useForm();
-  useEffect(() => document.title = props.title, [props.title]);
```
Comment (nk6432) [inline integrated-scouting-apps/src/routes/pit.tsx:11]: keep this

**Review by nk6432 — CHANGES_REQUESTED** [review]

File: integrated-scouting-apps/src/public/stylesheets/style.css (hunk 1 above)
Comment (rkang1357) [reply integrated-scouting-apps/src/public/stylesheets/style.css:31]: done

File: integrated-scouting-apps/src/public/stylesheets/style.css (hunk 2 above)
Comment (rkang1357) [reply integrated-scouting-apps/src/public/stylesheets/style.css:165]: done

File: integrated-scouting-apps/src/public/stylesheets/style.css (hunk 3 above)
Comment (rkang1357) [reply integrated-scouting-apps/src/public/stylesheets/style.css:173]: done

File: integrated-scouting-apps/src/routes/pit.tsx (hunk 5 above)
Comment (rkang1357) [reply integrated-scouting-apps/src/routes/pit.tsx:14]: done

File: integrated-scouting-apps/src/routes/pit.tsx (hunk 6 above)
Comment (rkang1357) [reply integrated-scouting-apps/src/routes/pit.tsx:16]: i need to import button in order to use buttons

File: integrated-scouting-apps/src/routes/pit.tsx (hunk 7 above)
Comment (rkang1357) [reply integrated-scouting-apps/src/routes/pit.tsx:11]: done

**Pull request closed without merging**